quanta-d51-2u/templates/flash_bios.sh

```diff
@@ -46,3 +46,6 @@ curl -T ./${BACKUP_FILE} <%= api.files %>/<%= nodeId %>-${VERSION}
 # Flash new image
 ${CMD} <%= sku.biosFirmware.args %>
 
+# Wait some time for the internal process to finish
+# otherwise catalog ami task after flashing will hang the node in some fw version (like 3A19)
+sleep 30
```

Comment:
I am afraid they are not the same case here. In BMC we can poll the status through ipmitool, while in BIOS, if we issue catalog command right after flashing, the node would hang and lose communication with RackHD. So in this case, a sleep without interacting with afulnx64 is necessary.